### ISSUE TITLE
RFIA Bill License

### ISSUE THREAD

**Issue by r-angeles** [responsible-financial-innovation-act22/RFIA-bill#96]
To follow through with the FOSS philosophy, I believe the project should choose a license as suggested in #15. As most know, without a license, the project would fall under default copyright laws as read in [Github's Licensing Page](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#choosing-the-right-license=):
> You're under no obligation to choose a license. However, without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work. If you're creating an open source project, we strongly encourage you to include an open source license. The [Open Source Guide](https://opensource.guide/legal/#which-open-source-license-is-appropriate-for-my-project) provides additional guidance on choosing the correct license for your project.

To stay true to the spirit of crypto, I nominate that the bill uses the same license that [Bitcoin's repo](https://github.com/bitcoin/bitcoin/blob/master/COPYING) uses. [The MIT License](https://opensource.org/licenses/MIT). Then, the RFIA Bill can be forked onto other projects and stored immutably permanently in the blockchain. **With this, the first-ever bill to ever exist in the blockchain will be born.**

This would result in a cascade. Everything will be stored in the blockchain immutably, birth certificates, medical certificates, credit card transactions, everything.

**Comment by NathanPeake** [comment]
They should pick a closed source government license.

**Comment by steakknife** [comment]
[Most U.S. government creative works such as writing or images are copyright-free. ](https://www.usa.gov/government-works)

I'm unsure if this is a personal or government work to assert this repo is definitely copyright-free.

**Comment by amcgregor** [comment]
@steakknife Here on GitHub, the agreement has explicit provisions. Any other declaration must be… declared. Additionally, and this is a quirky aspect: "public domain" does not exist worldwide. Ref: [Public Domain §8.3 Government Works](https://en.wikipedia.org/wiki/Public_domain#Government_works)

**Comment by steakknife** [comment]
TL;DR: I'm all for open government, if that is what this is, but not if
it's licensed differently (in the US, which is what matters, not Seychelles
or Tuvalu) as it could set a Damoclitic precedent.

I would consult a US government intellectual property lawyer
with experience in open source.

"Most U.S. government creative works such as writing or images are
copyright-free. But not everything is." https://www.usa.gov/government-works

If this was NOT created using government equipment, government time, and/or
government resources, then it may be a private work amenable to the MIT
license. However, a US government representative may either be conducting
activities as part of their duty or as a private party. The important point
is the content matter suggests it DOES materiality relate to their official
duties (i.e., you can't write a book for profit or manage a foundation on
government time). A conservative approach would be to license it exactly as
any other government work.

It would be a shame if open government activities, advanced somewhat in the
Obama years, were shut down for something problematic like a show-stopping
legal gotcha related to using public resources on something appearing to be
a nonprofit, private work. Keeping above board by consulting a subject
matter expert lawyer, i.e., in-house senate counsel, would be a smart move.
